contracts/PermissionedExchange.sol

```diff
@@ -178,6 +203,9 @@ contract PermissionedExchange is Initializable, OwnableUpgradeable {
         if (order.tokenGet == settings.getSQToken()) {
             require(tradeQuota[order.tokenGet][msg.sender] >= _amount, 'PE005');
         }
+        if (order.tokenGive == settings.getSQToken() && tradingLimitation > 0) {
```

Comment:
[question] unnecessary to require `tradingLimitation > 0`, if `tradingLimitation  == 0`, that means we don't allow user to purchase `kSQT`